### PR TITLE
fixup the api-approved annotation

### PR DESCRIPTION
**Pull request by astoycos**
The previous pr in our api-approved.kubernetes.io annotation did not include a review from the members of the `sig-network-api-approvers` group.  This PR is the original API implementation PR that was reviewed/approved by Tim Hockin, the core of the API hasn't really chaged since then. For future releases with major API changes we will ensure that the API is properly reviewed.

Please see https://github.com/kubernetes-sigs/network-policy-api/pull/143#issuecomment-1759926132 for more information on this

cc @thockin @liggitt

### CODE REVIEW

**Comment by astoycos** [comment]
/hold

Until Tim/Jordan look at this

**Comment by astoycos** [comment]
Poke @thockin @liggitt

**Comment by astoycos** [comment]
@liggitt we've spoken with the chairs and will not cut an official release without an ACK from them 👍

**Comment by tssurya** [comment]
@astoycos : Any updates on this one? I'd love a minor release since now nodes has merged and I plan on getting cidrs merged as well. If not I can continue using the commitID not a big deal.

**Comment by astoycos** [comment]
@tssurya If you could LGTM it I think we're good to go!!

**Review by tssurya — APPROVED** [review]
/lgtm

**Pull request merged**
